terraform/athena.tf

```diff
@@ -1,4 +1,147 @@
 resource "aws_athena_database" "s3_access_logs_db" {
   name   = "s3_access_logs_db"
   bucket = aws_ssm_parameter.athena_output_path.value
-  }
\ No newline at end of file
+  }
+
+  # AWS Glue Table
+resource "aws_glue_catalog_table" "prod_discover_s3_logs" {
+  name          = "prod_discover_s3_logs"
+  database_name = "s3_access_logs_db"
+
+  table_type = "EXTERNAL_TABLE"
+
+  parameters = {
+    EXTERNAL              = "TRUE"
+    "parquet.compression" = "SNAPPY"
+  }
+
+  storage_descriptor {
+    location      = "s3://pennsieve-prod-discover-publish-logs-use1/prod/discover-publish/s3"
```

Comment:
We should use Terraform variables to specify the AWS environment, and the S3 bucket name.